crates/schema-inference/src/server.rs

```diff
@@ -0,0 +1,220 @@
+use doc::inference::Shape;
```

Comment:
Does anyone know of a good tool to sort/organize these imports? I'd love something like [`import/order`](https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/order.md) from ESLint, to e.g put external imports above workspace imports, above crate imports, etc

Reply:
I'd advocate for using `rustmft` for this. We used to have it integrated into CI, and I think perhaps it just got lost at some point during a build refactor. But I think a sensible approach is to check in a `rustfmt.toml` and have CI just assert that everything is formatted correctly (and let devs wire up `rustfmt` into their workflows however they wish).

Reply:
Oh I see, rustfmt does this. I just had line breaks in my imports, so it was sorting them individually, not all together.